routes/parking.js

```diff
@@ -14,26 +66,68 @@ router.post('/create-parking', async (req, res) => {
     return res.status(400).send('Missing parameter')
   }
 
-  try {
-    const parking = new Parking({
-      name: name,
-      number: number,
-      rate: rate,
-      address: address
-    })
-    QRCode.toDataURL(parking.id, { width: 300 }, function (err, url) {
-      if (err) {
-        return res.status(500).send("failed to create qrcode");
-      }
-      // If qrcode generated successfully, we save the document for the space.
-      // console.log(url);
-      parking.qrCodeUrl = url;
-    })
+  const parking = new Parking({
+    name: name,
+    number: number,
+    rate: rate,
+    address: address,
+
+  })
+  QRCode.toDataURL(parking.id, { width: 300 }, function (err, url) {
+    if (err) {
+      return res.status(500).send("failed to create qrcode");
+    }
+    // If qrcode generated successfully, we save the document for the space.
+    parking.qrCodeUrl = url;
     parking.save()
-  } catch (err) {
-    console.log(err)
-    res.status(400).send(`create parking failed ${err}`)
+      .then(parking => {
+        addToCache(parking.toObject())
+        res.sendStatus(200)
+      })
+      .catch(err => res.status(400).send(`create parking failed ${err}`))
+  })
+})
+
+router.get('/all', async (req, res) => {
+  res.send(cachedParkingLots);
```

Comment:
you can just do `const parkingLots = Parking.find({ createdBy: userId})`

Reply:
Aren't all parking lots created by the same, hard-coded admin user id now?